migration/v2.27/README.md

```diff
@@ -109,8 +109,8 @@
     ```bash
     # Source credentials
     CK8S_KUBESPRAY_PATH=/path/to/compliantkubernetes-kubespray
-    terraform -chdir="${CK8S_KUBESPRAY_PATH}/kubespray/contrib/terraform/upcloud/" plan -var-file="${CK8S_CONFIG_PATH}/sc-config/cluster.tfvars" -state="${CK8S_CONFIG_PATH}/sc-config/terraform.tfstate" -var="inventory_file=${CK8S_CONFIG_PATH}/sc-config/inventory.ini"
-    terraform -chdir="${CK8S_KUBESPRAY_PATH}/kubespray/contrib/terraform/upcloud/" plan -var-file="${CK8S_CONFIG_PATH}/wc-config/cluster.tfvars" -state="${CK8S_CONFIG_PATH}/wc-config/terraform.tfstate" -var="inventory_file=${CK8S_CONFIG_PATH}/wc-config/inventory.ini"
+    terraform -chdir="${CK8S_KUBESPRAY_PATH}/kubespray/contrib/terraform/upcloud/" plan -var "UPCLOUD_PASSWORD=${UPCLOUD_PASSWORD}" -var "UPCLOUD_USERNAME=${UPCLOUD_USERNAME}" -var-file="${CK8S_CONFIG_PATH}/sc-config/cluster.tfvars" -state="${CK8S_CONFIG_PATH}/sc-config/terraform.tfstate" -var="inventory_file=${CK8S_CONFIG_PATH}/sc-config/inventory.ini"
+    terraform -chdir="${CK8S_KUBESPRAY_PATH}/kubespray/contrib/terraform/upcloud/" plan -var "UPCLOUD_PASSWORD=${UPCLOUD_PASSWORD}" -var "UPCLOUD_USERNAME=${UPCLOUD_USERNAME}" -var-file="${CK8S_CONFIG_PATH}/wc-config/cluster.tfvars" -state="${CK8S_CONFIG_PATH}/wc-config/terraform.tfstate" -var="inventory_file=${CK8S_CONFIG_PATH}/wc-config/inventory.ini"
```

Comment:
I think our typical process is to use the variable `TF_VAR_UPCLOUD_...=${UPCLOUD_...}`. Same as the upstream docs https://github.com/kubernetes-sigs/kubespray/tree/master/contrib/terraform/upcloud#quickstart

So I'd probably like us to continue using that. But I'd also like some more opinions here